webpack/webpack.config.js

```diff
@@ -1,13 +1,24 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin')
```

Comment:
@thet @erral I though an alternate solution using the standard page template and the `sed` command, but @rodfersou insisted on using this plugin here. I don't see a benefit with the plugin but I'm open to discuss it.

Reply:
I can't say anything about webpack configuration because I don't know how it works.

Reply:
@hvelarde keep in mind that this is not the most complex use case we have; this plugin is the way most of the JS projects with webpack work, and I see benefits when deal with more complex use cases.

Reply:
@rodfersou what are the benefits for this project?

Reply:
This is static resources we are talking.. the result page template will be the same, there is no overhead in the execution time.
You better than me know that javascript world changes really easy, and I prefer to keep myself learning at small steps.
If you prefer to do a different approach, try to use one that most people are using in JS and webpack world, not a sed soluction.
And I know what the other answer would be.. **html-load** plugin.  I used it some time ago but had to remove because it plays bad with css generation.